### PR TITLE
[WIP] Prep pipeline and deployments for shared secret process

### PR DESCRIPTION
**Pull request by wjwoodson**
Interpolate vars from file generated at runtime from secrets files instead of all the per-deployment silliness in https://github.com/18F/cg-deploy-bosh/pull/173

### CODE REVIEW

File: pipeline.yml [hunk 1]
```diff
@@ -868,13 +859,13 @@ resources:
...
-    region: {{aws-region}}
-    secrets_file: {{common-secrets-file}}
-    secrets_passphrase: {{common-secrets-secrets-passphrase}}
+#- name: secrets-master
+#  type: cg-common
+#  source:
+#    bucket_name: {{master-bucket-name}}
+#    region: {{aws-region}}
+#    secrets_file: {{master-secrets-file}}
+#    secrets_passphrase: {{master-secrets-secrets-passphrase}}
```
Comment (LinuxBozo) [inline pipeline.yml]: Any reason this is commented with the name change?

Comment (wjwoodson) [reply pipeline.yml]: masterbosh isn't using updated secrets process yet and the earlier `secrets-common` resource is no longer used across all deployments

Comment (LinuxBozo) [reply pipeline.yml]: Does that matter? A bucket, region, filename, and passphrase should all exist, yeah? Isn't this: https://github.com/18F/cg-secret-rotation/pull/6/files#diff-4e39b6ee47adc972ea04e30a02d2b1e2R152 And if not, why not? Also, this resource isn't referenced anywhere else in the pipeline as of yet.

Comment (wjwoodson) [reply pipeline.yml]: It is the same bucket, region, filename, and passphrase used in https://github.com/18F/cg-secret-rotation/blob/b7aaea2289a25ba5ae37694b7dbc51788a3410d8/ci/pipeline.yml#L217, only commented out _because_ `deploy-master-bosh` doesn't use secrets from there yet.

File: bosh-tooling-uaa.yml [hunk 2]
```diff
@@ -39,7 +39,7 @@ instance_groups:
             <<: *client-template
           # For authentication to bosh via concourse
           concourse_bosh:
-            secret: (( param "specify concourse bosh client secret" ))
+            secret: (( grab secrets.bosh_uaa_concourse_bosh_client_secret ))
```
Comment (LinuxBozo) [inline bosh-tooling-uaa.yml]: isn't this supposed to be `secrets.bosh_uaa_ci_client_secret` or am I missing something?

Comment (wjwoodson) [reply bosh-tooling-uaa.yml]: concourse uses a different client for interacting with tooling bosh https://github.com/18F/cg-deploy-bosh/pull/179/files#diff-688ee30f2b0e35b63848f09d2c9bbbb3R669 AFACT we should probably update to use `ci` like other deployments

**Pull request merged**